src/icon-manifest/languages.ts

```diff
@@ -16,6 +16,7 @@ export const languages: ILanguageCollection = {
   autoit: { ids: 'autoit', defaultExtension: 'au3' },
   avro: { ids: 'avro', defaultExtension: 'avcs' },
   azcli: { ids: 'azcli', defaultExtension: 'azcli' },
+  azure_pipelines: { ids: 'azure-pipelines', defaultExtension: 'azure-pipelines.yml' },
```

Comment:
Can you point us to the `vscode` extension that uses `azure-pipelines` as `languageId`?

Reply:
It hasn't shipped yet, and the repo isn't public. I can add you or someone else who needs to verify: https://github.com/Microsoft/azure-pipelines-vscode

If you want to wait until it ships before merging this PR, I'm 👌 with that. I expect to ship it imminently.

Reply:
Nope. We are fine to early support it. Thanks for letting us now.

Reply:
(Repo will be public when we ship!)